src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsForm/TransformationsForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+
+import { TransformationField } from '../TransformationsField';
+
+const TransformationsForm = ({ handleSubmit }) => {
+  return (
+    <form onSubmit={handleSubmit}>
+      <TransformationField />
+    </form>
+  );
+};
+
+export default stripesFinalForm({
```

Comment:
I suggest using the named export to do less stuff in the future. (As it was done in stripes-data-transfer-components)

Reply:
Good catch, yep. I used named for another form and forgot to use it here. Yeah, let's gradually adopt named exports pattern.